Bottom: bb9ef433599ef5f4db95e5c3c50c3b2c87c005f4
Top:    360ef547384c54ed85ec0365e326c6e67d609221
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:07:58 -0500

Refresh of add-haproxy-role

---
diff --git a/ansible-playbook/group_vars/docker-hosts/networking.yml b/ansible-playbook/group_vars/docker-hosts/networking.yml
index 7d353b98a..32d99802b 100644
--- a/ansible-playbook/group_vars/docker-hosts/networking.yml
+++ b/ansible-playbook/group_vars/docker-hosts/networking.yml
@@ -22,12 +22,14 @@ testssh_port_args:
 
 exim4_hostname: mail.novatech-llc.com
 exim4_port_args:
-- "25"
+- "25:25"
+- "465:465"
+- "587:587"
 
 openldap_hostname: ldap.novatech-llc.com
 openldap_port_args:
-- "{{ openldap_ip_addr }}:389:389"
-- "{{ openldap_ip_addr }}:636:636"
+- "389:389"
+- "636:636"
 
 phpldapadmin_hostname: ldap.novatech-llc.com
 phpldapadmin_port_args: []
@@ -36,7 +38,7 @@ phpmyadmin_hostname: phpmyadmin.novatech-llc.com
 
 buildbot_hostname: buildbot.novatech-llc.com
 buildbot_port_args:
-- "{{ buildbot_ip_addr }}:9989:9989"
+- "9989:9989"
 
 nexus_hostname: nexus.novatech-llc.com
 nexus_port_args: []
